src/index.html

```diff
@@ -8,6 +8,42 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
```

Comment:
```suggestion
    <div class="card" data-qa="card">
```

We put attributes in new line if element has long attribute values or number of attributes is more than 2